SourceX/miniwin/misc_msg.cpp

```diff
@@ -264,7 +264,7 @@ bool false_avail(const char *name, int value)
 {
 #ifndef __vita__
 	// Logging on Vita is slow due slow IO, so disable spamming unhandled events to log
-	SDL_Log("Unhandled SDL event: %s %d", name, value);
+	SDL_LogDebug(SDL_LOG_CATEGORY_APPLICATION, "Unhandled SDL event: %s %d", name, value);

```

Comment:
The requirement to add categories when using the other log levels is the primary reason I haven't been making use of them.

Would it make sense for us to make some short hand functions?

`SDL_LogDebug("Unhandled SDL event: %s %d", name, value);`

Reply:
We can make shorthand functions separately without the `SDL_` prefix.
This is out-of-scope for this PR, which is mainly concerned with backporting.

Reply:
Agreed, and sounds good :+1: